cdap-watchdog-api/src/main/java/io/cdap/cdap/api/metrics/MetricValue.java

```diff
@@ -15,19 +15,77 @@
  */
 package io.cdap.cdap.api.metrics;
 
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
Don't use sun.reflect. Standard is `UnsupportedOperationException` that is standard API

Reply:
Changed.